lib/dl_api_commons/pyproject.toml

```diff
@@ -18,8 +18,7 @@ marshmallow = ">=3.19.0"
 multidict = ">=4.0"
 opentracing = ">=2.4.0"
 python = ">=3.10, <3.12"
-raven = ">=6.10.0"
-sentry-sdk = ">=1.15.0"
+sentry-sdk = {extras = ["flask"], version = ">=1.15.0"}
```

Comment:
NIT: can be changed to `sentry-sdk = {version = ">=1.15.0", extras = ["flask"]}` for easier reading